python/orca/src/bigdl/orca/learn/tf2/tf_runner.py

```diff
@@ -59,7 +59,7 @@ def find_free_port():
 def _try_import_strategy():
     """Late import for TensorFlow"""
     import tensorflow as tf
-    return tf.distribute.experimental.MultiWorkerMirroredStrategy
+    return tf.distribute.MultiWorkerMirroredStrategy
```

Comment:
`tf.distribute.experimental.MultiWorkerMirroredStrategy` this is now deprecated, remove the `experimental`